src/Parser/Monk/Windwalker/Modules/Items/TheEmperorsCapacitor.js

```diff
@@ -65,6 +68,21 @@ class TheEmperorsCapacitor extends Analyzer {
     }
   }
 
+  _isPermanentPet(guid) {
+      return Object.keys(PETS).map(key => PETS[key]).every(pet => pet.id !== guid);
+  }
+
+  on_byPlayerPet_damage(event) {
+      const pet = this.pets.getSourceEntity(event);
+      const spellId = event.ability.guid;
+      if (!this._isPermanentPet(pet.guid)) {
+         return;
```

Comment:
Can you explain why you go to the trouble of excluding some pets here? Does any pet other than SEF clones use CJL? If another pet used CJL wouldn't you want it included?